frontend/src/metabase/css/core/colors.module.css

```diff
@@ -56,8 +56,8 @@
   );
   --mb-color-success: #84bb4c;
   --mb-color-summarize: #88bf4d;
-  --mb-color-error: #ed6e6e;
-  --mb-color-danger: #ed6e6e;
+  --mb-color-error: var(--mb-base-color-lobster-50);
```

Comment:
new colors are grabbed from [this design.](https://www.figma.com/design/W1TfzdrR8ldzp4vdjP1z36/SDK-Embedding%3A-admin-settings-%2B-embed-flow?node-id=11613-23031&t=z4vjXKUradcAyyAp-0)